samples/nrf_desktop/configuration/app_nrf52840_pca10059.conf

```diff
@@ -11,6 +11,8 @@ CONFIG_DESKTOP_USB_ENABLE=y
 CONFIG_DESKTOP_BLE_SCANNING_ENABLE=y
 CONFIG_DESKTOP_BLE_SCANNING_DEVICE_NAME="Mouse nRF52 Desktop"
 
+CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE=y
```

Comment:
Already did the clean up in PR https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/362, together with other desktop/NCS options cleanup.